tests/command.bats

```diff
@@ -1306,3 +1306,53 @@ EOF
 
   unstub docker
 }
+
+export BUILDKITE_AGENT_STUB_DEBUG=/dev/tty
```

Comment:
this should be part of the comment at the top of the file (and commented out :P)